### ISSUE TITLE
highlight broadcasts on statistic page that the user supports

### ISSUE THREAD

**Issue by davidak** [roschaefer/rundfunk-mitbestimmen#427]
* **I'm submitting a ...**
  * user interface improvement

* **What is the current behavior?**
plain table

* **What is the expected behavior?**
table with highlighted broadcasts

also option to support currently not supported broadcast
and desupport supported broadcasts

of course newly supported broadcasts should also be highlighted
and vice versa

* **What is the motivation / use case for changing the behavior?**
i was clicking through the stats and found some bradcasts i like, so i wanted to support them. currently i have to search for them on another site...

**Comment by roschaefer** [comment]
Thank you @davidak for submitting this issue. I updated the description, is this what you want?

**Comment by roschaefer** [comment]
Being able to support a broadcast immediately would be nice, but there is a small problem: We track if a user as "seen" a broadcast, i.e. we save an "impression" in the database. This way we can compare how many people have seen the broadcast and did *not* click on "support". It is necessary to overcome unfair biases - a brand new broadcast has 0 supporters, but that's not because it's unpopular. It's because it was just added to the database.

So, we will probably not add a support button if the description is not visible (which is the case on the statistic page), but we can make the title of the broadcast a link to the broadcast page. A user can click on the title, visit the broadcast page `broadcast/4711` and click support. That *is* a context switch. But at least you have a link on the statistics page, you see the description and the system can save an impression.

**Comment by davidak** [comment]
Yes, but with "highlight" i mean that the row should have a different background, like green or a darker one. But might not be necesarry.

**Comment by roschaefer** [comment]
Maybe a heart icon is unambiguous? A heart or a thumbs up icon means sth. like "I like that" whereas a green background means "good". 

If there is enough space, we could also add the respective amount of the supported broadcast in green next to the total amount.

E.g.
```
164.56€ (+0.40€)
```

**Comment by davidak** [comment]
Since i thought about it longer i wouldn't color the rows. It's too much color. The heart would be enough to stand out. But when there are technical limitations, making the title a link to the detail page is also a good solution.

**Comment by roschaefer** [comment]
I would say, let's do both. I'm already creating a PR with links to the broadcast pages...

**Comment by davidak** [comment]
Adding your personal contribution to the amount is also very good. That might be enough to highlight broadcasts a user supports. Also a button to edit the amount, maybe a gear sign?

**Comment by davidak** [comment]
Right, do it! :)

**Comment by roschaefer** [comment]
Editing the amount is a bit tricky, too. The system distributes your budget evenly. So you should edit the amount on a page where you can see all your amounts. E.g. If you fixed an amount to `10€` of one broadcast, you cannot give another broadcast `8€`, that would be `10€+8€ = 18€ > 17.50`. The statistic table is too large to see all your amounts at once.

**Comment by davidak** [comment]
What i mean is adding a gear icon that is a link to "my broadcasts". Maybe automatically select the text field of that broadcasts amount.

**Comment by roschaefer** [comment]
Oh, yep, that would be great. If you click on `(+0.40€)` you get to the "My broadcasts" page.
